modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a storage account that has public network access disabled and is exposed behind a private endpoint on Azure.
+
+.Prerequisites
+
+* You have configured an image registry to run on Azure
+* You must know the VNet and subnet names used for your Azure environment.
+* You have configured a route
+
+.Procedure 
+
+. Enter the following command to obtain the VNet and subnet names that are used in the cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -nopenshift-machine-api -ojson|jq '.items[1].spec.template.spec.providerSpec.value | (.vnet, .subnet)'
+----
```

Comment:
Users should know their vnet and subnet names for this procedure, we should not tell them to get it from machinesets (they might even get the wrong names from this command)

```suggestion

```